pages/calldata-decoder.test.tsx

```diff
@@ -187,4 +189,73 @@ describe(CalldataDecoder.name, () => {
       decodedCalldataTree1.querySelector('#node-type')?.innerHTML!,
     ).toEqual(expect.stringMatching('address'));
   });
+
+  it.skip('types wrong abi, gets error message', async () => {
```

Comment:
why skip?

Reply:
This test would currently fail. I've decided to write it as evidence that we lack support for this type of ABI. I will add support for it in the next PR.